en/docs/learn/examples/rest-api-examples/configuring-non-http-endpoints.md

```diff
@@ -10,18 +10,19 @@ Following is a sample REST API configuration that we can used to implement this
         <resource methods="POST" url-mapping="/"> 
            <inSequence> 
               <property name="REST_URL_POSTFIX" action="remove" scope="axis2"></property> 
-              <send> 
+              <call> 
```

Comment:
Shouldn't we set out_only true? With this does the request return?

Reply:
fix it